src/platforms/hosted/bmp_hosted.h

```diff
@@ -48,11 +48,12 @@ struct trans_ctx {
 	volatile unsigned long flags;
 };
 
-typedef struct usb_link_s {
+typedef struct usb_link {
```

Comment:
`usb_link_t` -> `usb_link_s`
though I'm guessing that's on fix/naming-conformity